ens/src/utils.ts

```diff
@@ -134,4 +136,35 @@ export function byteArrayFromHex(s: string): ByteArray {
 export function uint256ToByteArray(i: BigInt): ByteArray {
   let hex = i.toHex().slice(2).padStart(64, '0')
   return byteArrayFromHex(hex)
+}
+
+export function decodeName(buf: Bytes, txHash: string): Array<string> | null {
```

Comment:
Suggesion: can we have separate test case for utils functions?,this function seems major

Reply:
added test for decode name

Reply:
done